images/virtualization-artifact/pkg/controller/vd/internal/life_cycle.go

```diff
@@ -31,20 +32,24 @@ import (
 )
 
 type LifeCycleHandler struct {
+	logger  *slog.Logger
 	client  client.Client
 	blank   source.Handler
 	sources *source.Sources
 }
 
-func NewLifeCycleHandler(blank source.Handler, sources *source.Sources, client client.Client) *LifeCycleHandler {
+func NewLifeCycleHandler(logger *slog.Logger, blank source.Handler, sources *source.Sources, client client.Client) *LifeCycleHandler {
 	return &LifeCycleHandler{
+		logger:  logger,
 		client:  client,
 		blank:   blank,
 		sources: sources,
 	}
 }
 
 func (h LifeCycleHandler) Handle(ctx context.Context, vd *virtv2.VirtualDisk) (reconcile.Result, error) {
+	logger := h.logger.With("vdName", vd.Name, "vdNamespace", vd.Namespace)
```

Comment:
Why "vdName" and not "name"? Why do you need the prefix "vd". The log contains the name of the controller that processes the resource, right?